ocaml/xapi/network_event_loop.ml

```diff
@@ -0,0 +1,92 @@
+module D=Debug.Make(struct let name="network_event_loop" end)
+
+module StringMap = Map.Make(struct type t = string let compare = compare end)
+
+type 'a update = Deleted | Snapshot of 'a
```

Comment:
I don't think you're using this any more

Reply:
Oops thanks, I forgot to delete that.

Reply:
Using it on line 41 of the first commit, but perhaps it goes away later...

Reply:
Yep. It's better to just check the whole diff.